practice_app/api/tests.py

```diff
@@ -101,4 +101,37 @@ def test_valid_rows(self):
         self.assertContains(response, 'source')
         self.assertContains(response, 'date')
         self.assertContains(response, 'url')
-        self.assertContains(response, 'position')
\ No newline at end of file
+        self.assertContains(response, 'position')
+class zenodo_test_cases(TestCase):
+
+    def setUp(self):
+        self.c = Client()
+
+    def tearDown(self):
+        print('GET Tests Completed Successfully')
+
+    def test_zenodo_api(self):
+        ACCESS_TOKEN = api_keys.api_keys['zenodo_api']
+        request = requests.get('https://zenodo.org/api/records',
```

Comment:
Why are you making this call here? 

To me, you don't need to test Zenodo api in your unit test. Since you are testing your api call and it wraps Zenodo api functionality, you just need to test the response of your api call. 

Please correct me if I'm missing something.